apis/Google.Cloud.Spanner.Admin.Database.V1/Google.Cloud.Spanner.Admin.Database.V1/DatabaseAdminClient.g.cs

```diff
@@ -404,16 +404,33 @@ public sealed partial class DatabaseAdminClientBuilder : gaxgrpc::ClientBuilderB
         /// <summary>The settings to use for RPCs, or <c>null</c> for the default settings.</summary>
         public DatabaseAdminSettings Settings { get; set; }
 
+        partial void PartialBuild(ref DatabaseAdminClient client);
```

Comment:
nit: OnBuild?

Reply:
Maybe. It's trying to act as a potential replacement rather than just a response to the existing code though. We can definitely bike shed on names when we're all happy with the overall idea :)